app/src/test/java/pm/gnosis/heimdall/Test.kt

```diff
@@ -0,0 +1,62 @@
+package pm.gnosis.heimdall
+
+import com.squareup.moshi.*
+import org.junit.Assert
+import org.junit.Test
+import pm.gnosis.crypto.KeyPair
+import pm.gnosis.crypto.utils.Sha3Utils
+import pm.gnosis.heimdall.data.remote.models.push.PushMessage
+import pm.gnosis.heimdall.data.remote.models.push.SendTransactionHashParams
+import java.math.BigInteger
+
+
+class Test {
```

Comment:
Can we find a better name or delete it :D

Reply:
Will delete it